src/components/Goods.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Goods.scss';
+
+export class Goods extends React.Component {
+  state = {
+    goods: [...this.props.goods],
```

Comment:
things to remember about passing props to state

[https://stackoverflow.com/a/50403930](https://stackoverflow.com/a/50403930)

[https://reactjs.org/docs/thinking-in-react.html#step-3-identify-the-minimal-but-complete-representation-of-ui-state](https://reactjs.org/docs/thinking-in-react.html#step-3-identify-the-minimal-but-complete-representation-of-ui-state)

[https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html)